class Error(Exception):
    pass

class RecordError(Error):
    pass
